schema/colors.json

```diff
@@ -0,0 +1,107 @@
+[
```

Comment:
Is this file specific to k8s logs and if so should we name it accordingly?

Reply:
Changed name and moved from the schema folder as this is not related to schemas. If you have a better idea for naming it please let me know, or feel free to edit it yourself :+1: